.travis.yml

```diff
@@ -1,60 +0,0 @@
-language: php
```

Comment:
Could you please remove this file from `.gitattributes` aswell?

Reply:
> Maybe we should drop the whole CI stuff at all as there is already a PR #18 which introduces the new workflow.
> So you can drop the commits which are related to the workflow with `git rebase`.

I checked the PR #18, drop the whole CI stuff commit is done.



> @gpgkd906 #18 is almost ready to merge. After this, you might want to rebase against 3.1.x branch of the laminas package.

I got it.